From ce83ae6fa07d449eeae29f4828bfde4e64b06e9c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Rene=CC=81=20de=20Hesselle?= <dehesselle@web.de>
Date: Mon, 27 Mar 2023 01:10:24 +0200
Subject: [PATCH] Remove gtk-mac-integration dependency

The only piece of code that required this has been removed with
commit 33b899247d06529b756a2a75f60a8e14e9adf115.
---
 configure.ac        | 7 -------
 gspell/gspell-osx.c | 1 -
 2 files changed, 8 deletions(-)

diff --git a/configure.ac b/configure.ac
index f245622..2d9609f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -193,13 +193,6 @@ AM_CONDITIONAL(OS_OSX, test "$os_osx" = "yes")
 
 if test "$os_osx" = "yes"; then
 	AC_DEFINE([OS_OSX], [1], [Defined if OS is Mac OSX])
-
-	AX_PKG_CHECK_MODULES([GTK_MAC],
-			     [],
-			     [gtk-mac-integration-gtk3 >= 2.0.8])
-
-	AC_SUBST(GTK_MAC_CFLAGS)
-	AC_SUBST(GTK_MAC_LIBS)
 fi
 
 # Output files
diff --git a/gspell/gspell-osx.c b/gspell/gspell-osx.c
index 93aaa75..fa08421 100644
--- a/gspell/gspell-osx.c
+++ b/gspell/gspell-osx.c
@@ -22,7 +22,6 @@
 #endif
 
 #include "gspell-osx.h"
-#include <gtkosxapplication.h>
 #import <Cocoa/Cocoa.h>
 
 gchar *
-- 
2.37.1 (Apple Git-137.1)

